wcmc_components/app/controllers/wcmc_components/api_controller.rb

```diff
@@ -0,0 +1,3 @@
+module WcmcComponents
```

Comment:
Can this file be deleted, or is it being used in some behind the scenes railsy way?